projects/core/styles/variables/media.less

```diff
@@ -21,3 +21,9 @@ next min-width - 60% from 1px (1/16 * 0.6 = 0.0375)
 @tui-desktop-interval: ~'(min-width: 64em) and (max-width: 79.9625em)';
 
 @tui-desktop-lg-min: ~'screen and (min-width: 80em)';
+
+@tui-touch: ~'(hover: none) and (pointer: coarse)';
+@tui-stylus: ~'(hover: none) and (pointer: fine)';
+@tui-pointer: ~'(hover: hover) and (pointer: coarse)';
+@tui-mouse: ~'(hover: hover) and (pointer: fine)';
```

Comment:
I'm pretty sure we have some places in code where we only check for `(hover: hover)`. Let's replace them all with `@tui-mouse`, I don't think we ever need to have hover effects for coarse pointer.

Reply:
fixed